src/datamap.jsx

```diff
@@ -77,6 +78,7 @@ export default class Datamap extends React.Component {
 			data,
 			graticule,
 			labels,
+            legends,
```

Comment:
This looks to be a typo.

```suggestion
			legend,
```